### PR TITLE
Implement possibility to add additional ssh keys

### PR DESCRIPTION
**Pull request by Audifire**
This pull request implements the possibility, to add existing ssh keys from Hetzner Cloud.
This is very useful, if you want to use your already existing private key to login via SSH to the vServers.

### CODE REVIEW

File: component/component.js [hunk 1]
```diff
@@ -115,6 +126,13 @@ export default Ember.Component.extend(NodeDriver, {
       let options = [...select.target.options].filter(o => o.selected).map(o => o.value)
       this.set('model.%%DRIVERNAME%%Config.networks', options);
     },
+    modifyKeys: function (select) {
+      let options = [...select.target.options]
+        .filter(o => o.selected)
+        .map(o => o.value)
+        .map(o => this.keyChoices.filter(keyChoice => keyChoice.id == o)[0]["public_key"]);
```
Comment (mxschmitt) [inline component/component.js:133]:
```suggestion
        .map(o => this.keyChoices.find(keyChoice => keyChoice.id == o.value)["public_key"]);
```

**Review by mxschmitt — APPROVED** [review]
Other than that it looks good to me, thanks!

**Comment by Audifire** [comment]
Now I applied your suggestion and saw, there is a merge conflict.

So I solved the conflict with rebasing my branch to master again. But the new version is not displayed here.

Maybe I have to reopen the PR again, that it's showing correctly. I've done that in Gitlab already and there it's working perfectly. I don't know why it's not working here.

**Review by mxschmitt — APPROVED** [review]

**Comment by mxschmitt** [comment]
sorry, for the late response, will merge now.

**Pull request merged**